tests/zfs-tests/cmd/randfree_file/randfree_file.c

```diff
@@ -89,10 +90,12 @@ main(int argc, char *argv[])
 	}
 
 	buf = (char *)malloc(filesize);
+	memset(buf, 0x0, filesize);
```

Comment:
Normally I'd suggest using `calloc()` here since you'll get an optimized version which is already zero filled.  But instead I think writing a known pattern in better in this case.  How about `memset(buf, 'a', filesize)`.